app/src/main/java/de/danoeh/antennapod/fragment/CompletedDownloadsFragment.java

```diff
@@ -322,6 +366,28 @@ public void onEndSelectMode() {
         swipeActions.attachTo(recyclerView);
     }
 
+    /**
+     * Returns the sort order for the downloads.
+     */
+    public SortOrder getDownloadsSortedOrder() {
+        String sortOrderStr = prefs.getString(PREF_DOWNLOADS_SORTED_ORDER, "use-default");
+        Log.d(TAG, sortOrderStr);
+        return SortOrder.parseWithDefault(sortOrderStr, SortOrder.DATE_NEW_OLD);
```

Comment:
I know that this is also done for the queue, but the following would be much more simple (that could maybe also be changed for the queue, so we can delete `parseWithDefault`:

```suggestion
        String sortOrderStr = prefs.getString(PREF_DOWNLOADS_SORTED_ORDER, "" + SortOrder.DATE_NEW_OLD.code);
        return SortOrder.fromCodeString(sortOrderStr);
```

Using `.code` instead of `.name()` also makes it more "stable", such that there are no surprises if in the future someone renames the enum. That's too late for the queue already, but we can at least do it right here :)